sql/core/src/test/scala/org/apache/spark/sql/CachedTableSuite.scala

```diff
@@ -985,4 +976,59 @@ class CachedTableSuite extends QueryTest with SQLTestUtils with SharedSQLContext
     val queryStats3 = query().queryExecution.optimizedPlan.stats.attributeStats
     assert(queryStats3.map(_._1.name).toSet === Set("c0", "v1", "v2"))
   }
+
+  test("Refresh Qualified Tables") {
```

Comment:
Since this is a bug fix, please add `SPARK-27248` prefix to all test cases.
```scala
- test("Refresh Qualified Tables") {
+ test("SPARK-27248 Refresh Qualified Tables") {
```

Reply:
Thanks. Just updated.